hacks/cypress_env_local.sh

```diff
@@ -14,10 +14,12 @@ export CYPRESS_BASE_URL=http://localhost:3000
 export CYPRESS_SSH_PUB_KEY=$(cat ~/.ssh/id_rsa.pub)
 
 export CYPRESS_CLUSTER_NAME=test-infra-cluster-assisted-installer
+export CYPRESS_CLUSTER_BASE_DOMAIN='redhat.com'
 export CYPRESS_CLUSTER_HOSTNAME_MASTER_PREFIX=test-infra-cluster-assisted-installer-master
 export CYPRESS_CLUSTER_HOSTNAME_WORKER_PREFIX=test-infra-cluster-assisted-installer-worker
 export CYPRESS_PULL_SECRET=$(cat ~/pull-secret)
 export CYPRESS_ISO_PATTERN="~/Downloads/cluster-*-discovery.iso"
+export CYPRESS_OPENSHIFT_VERSION='OpenShift 4.6.8'
```

Comment:
You could use just '4.7', so that we pass the value not the display value. Also that would spare us on updating on each new release.

Reply:
Done. Aligned with your code from PR 96 downstream.